source/Octopus.Cli/Commands/RunbookRun/ListRunbookRunsCommand.cs

```diff
@@ -0,0 +1,111 @@
+                commandOutputProvider.Information("Did not find any runbook runs matching the search criteria.");
+
+            commandOutputProvider.Debug($"Showing {runbookRuns.Count} results...");
+
+            foreach(var item in runbookRuns) {
+                LogrunbookRunInfo(commandOutputProvider, item);
+            }
+
+            if (numberOfResults.HasValue && numberOfResults != runbookRuns.Count)
+                commandOutputProvider.Debug($"Please note you asked for {numberOfResults} results, but there were only {runbookRuns.Count} that matched your criteria");
+        }
+
+        public void PrintJsonOutput()
+        {
+            commandOutputProvider.Json(
+                runbookRuns.Select(run => new
+                {
+                    Id = run.Id,
+                    Name = run.Name,
+                    Created = run.Created,
+                    Project = new { Id = run.ProjectId, Name = projectsById[run.ProjectId].Name},
+                    Runbook = new { Id = run.RunbookId, Name = runbooksById[run.RunbookId].Name},
+                    Environment = new { Id = run.EnvironmentId, Name = environmentsById[run.EnvironmentId].Name},
+                    Tenant = new { Id = run.TenantId, Name = !string.IsNullOrEmpty(run.TenantId) ? tenantsById[run.TenantId].Name : null },
+                    FailureEncountered = run.FailureEncountered,
+                    Comments = run.Comments
+                }));
+        }
+
+        private void LogrunbookRunInfo(ICommandOutputProvider outputProvider,
+            RunbookRunResource runbookRunItem)
+        {
+            var nameOfrunbookRunEnvironment = environmentsById[runbookRunItem.EnvironmentId].Name;
+            var nameOfrunbookRunProject = projectsById[runbookRunItem.ProjectId].Name;
+            var nameOfrunbook = runbooksById[runbookRunItem.RunbookId].Name;
+
+            outputProvider.Information(" - Id: {Name:1}", runbookRunItem.Id);
+            outputProvider.Information(" - Name: {Name:1}", runbookRunItem.Name);
+            outputProvider.Information(" - Project: {Project:l}", nameOfrunbookRunProject);
+            outputProvider.Information(" - Runbook: {Runbook:l}", nameOfrunbook);
+            outputProvider.Information(" - Environment: {Environment:l}", nameOfrunbookRunEnvironment);
+
+            if (!string.IsNullOrEmpty(runbookRunItem.TenantId))
+            {
+                var nameOfrunbookRunTenant = tenantsById[runbookRunItem.TenantId].Name;
+                outputProvider.Information(" - Tenant: {Tenant:l}", nameOfrunbookRunTenant);
+            }
+
+            outputProvider.Information("\tCreated: {$Date:l}", runbookRunItem.Created);
+            if (!string.IsNullOrWhiteSpace(runbookRunItem.Comments)) outputProvider.Information("\tComments: {$Comments:l}", runbookRunItem.Comments);
+            outputProvider.Information("\tFaulure Encountered: {FailureEncountered:l}", runbookRunItem.FailureEncountered ? "Yes" : "No");
```

Comment:
Replace `Faulure` with `Failure`

Reply:
Typo fixed. You have a good eye for detail 👍